src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1047,11 +1047,6 @@ private async ValueTask<Http2Stream> SendHeadersAsync(HttpRequestMessage request
                 // Throw a retryable request exception if this is not result of some other error.
                 // This will cause retry logic to kick in and perform another connection attempt.
                 // The user should never see this exception.  Same logic lives in AddStream.
-                if (_abortException != null)
-                {
-                    ExceptionDispatchInfo.Throw(_abortException);
-                }
```

Comment:
If we're going to delete this, then the comment needs to be updated as well.  In https://github.com/dotnet/corefx/pull/39135/files#diff-7a94ca36fa7e17540e8c66bc9491a6e1R1052 I preferred to for now throw a wrapper exception, figuring we could separately and subsequently determine based on @wfurt's response whether retrying here is appropriate or not.